transactions.go

```diff
@@ -67,8 +66,6 @@ func (b *BuxClient) SendToRecipients(ctx context.Context, recipients []*transpor
 	draft, err := b.DraftToRecipients(ctx, recipients, metadata)
 	if err != nil {
 		return nil, err
-	} else if draft == nil {
```

Comment:
Why this has been removed ?

Reply:
Because it's being checked in the method `b.DraftToRecipients` :)